### DISCUSSION TITLE
Bug: incorrect pull request content shown in a PR

### DISCUSSION THREAD

**Discussion by ForNeVeR** [community/community#61985] [Pull Requests]
### Select Topic Area

Bug

### Body

See [this pull request](https://github.com/ForNeVeR/bloodmasters/pull/76).

If you open the **Files changed** tab, you'll see that `OnClosing` has been renamed to `OnFormClosing`, which is correct.
<img width="939" alt="image" src="https://github.com/community/community/assets/92793/12de119f-d42f-4ff0-925b-8fdeb3356cb2">

But what GitHub doesn't show is that the call to that method below has also been changed! Compare the full PR content:
<img width="949" alt="image" src="https://github.com/community/community/assets/92793/3a9f0ce7-4622-4bcb-b73c-e7e59bbf42e7">

with an actual file text from [the latest commit from that PR](https://github.com/KeterSCP/bloodmasters/commit/2aca1e9ccf4b59fc6bc72180da74c720f2daf143):
<img width="815" alt="image" src="https://github.com/community/community/assets/92793/3cd00288-c07a-4c08-a5c2-63c22d4fc2b0">

To me it looks like the PR wasn't properly updated after a force-push by GitHub. But it was… partially updated?

**Comment by ForNeVeR** [comment]
Another weird thing with same PR: after I've merged it into `main`, it wasn't marked as merged by GitHub. Here's a screenshot of my repo's history, showing `2aca1e9ccf4b59fc6bc72180da74c720f2daf143` as the top commit:
<img width="950" alt="image" src="https://github.com/community/community/assets/92793/f69e557f-0193-4827-b917-b0dff5478469">

Also at the same time the PR is shown as open, holding the same commit:
<img width="688" alt="image" src="https://github.com/community/community/assets/92793/a594f61c-a5d1-41b8-8b4e-e9e0509b1885">

I had to close it manually.

**Comment by undrcrxwn** [comment]
привет рустам
